client/src/pages/Profile.jsx

```diff
@@ -130,22 +134,15 @@ export default function Profile() {
 
         <SelectStatus value={selectStatus} onChange = {handleSelect}/>
 
-        <p>Comments</p>
-        <p>Notes</p>
+        <p onClick={()=> setActiveTab('comments')}>Comments</p>
+        <p onClick={()=> setActiveTab('notes')}>Notes</p>
         <button onClick={() => {setShow(true)}}>Add Note</button>
       </div>
 
 
+      
       {
-        show && 
-        <NotePopup 
-            show = {show}
-            onClose={() => {setShow(false)}}
-        />
-      }
-
-
-      {
+        (activeTab === 'books') &&
```

Comment:
Very nice use of conditional rendering! It's super clear and shows what tab is active.